.github/workflows/build.yml

```diff
@@ -201,9 +209,13 @@ jobs:
         with:
           # This functionality is poorly implemented and has been
           # causing problems due to the MITM implementation hogging or
-          # leaking memory.  If this happens to you just uncomment
-          # this line.
-          # monitor_permissions: "false"
+          # leaking memory.  As a result we disable it by default.  If
+          # you want to temporarily enable it, simply set
+          # monitor_permissions equal to "true".
+          #
+          # TODO: Re-enable this functionality when practical.  See
+          # cisagov/skeleton-generic#207 for more details.
```

Comment:
I created a separate, repo-specific issue for cases like this.

Reply:
Please see https://github.com/cisagov/ncats-webui/pull/51/commits/69b8c4b307005f028b8922813d5426dbaa407661.